controllers/admin/AdminPPreferencesController.php

```diff
@@ -225,6 +225,14 @@ public function __construct()
                         'required'   => false,
                         'type'       => 'bool',
                     ],
+                    'PS_SHOW_CONDITION'             => [
+                        'title'      => $this->l('Show condition'),
+                        'hint'       => $this->l('Show/Hide conditions dropdown list on product page.'),
```

Comment:
This hint talks about a "dropdown list", but AFAIK product condition is plain text on the product page.